### PR TITLE
Update languages.yaml

### PR DESCRIPTION
**Pull request by tyrion273**

### CODE REVIEW

File: languages.yaml [hunk 1]
```diff
@@ -206,14 +206,14 @@ hu:
...
+    USERNAME: Benutzername
     PASSWORD: Passwort
     ACCESS_DENIED: Zugang verweigert
     LOGIN_FAILED: Login fehlgeschlagen...
     LOGIN_SUCCESSFUL: Du wurdest erfolgreich eingeloggt.
-    BTN_LOGIN: Einloggen
+    BTN_LOGIN: Anmelden
     BTN_LOGOUT: Ausloggen
-    BTN_FORGOT: Vergessen
+    BTN_FORGOT: Passwort vergessen
```
Comment (Vivalldi) [inline languages.yaml]: I don't think this needs to be `Passwort vergessen` as non of the other languages state what's been forgot. In english it's `forgot` and not `Forgot Password`

**Comment by tyrion273** [comment]
Mainly for consistency as the admin plugin in german says Passwort vergessen.

**Comment by tyrion273** [comment]
Hello Vivalldi,

maybe as a compromise both times Vergessen as both times it is Forgot?

**Comment by Vivalldi** [comment]
Yeah, if you want to update your branch with `Vergessen` instead I think that'd work

**Comment by tyrion273** [comment]
Updated. And I suggested Vergessen on Crowdin!?

**Comment by Vivalldi** [comment]
:+1: thanks!

**Pull request merged**